src/data.ts

```diff
@@ -0,0 +1,310 @@
+export const lectures: Array<{
+  section: string
+  videoId: string
+  pages: Array<number>
+  subtitle: string
+}> = [
+  {
+    section: "1.1 Nomenclature",
+    videoId: "wbnTnBQNDr4",
+    pages: [1],
+    subtitle: "Filmed in Buenos Aires on May 10, 2024",
+  },
+  {
+    section: "1.2 Driving Factors",
+    videoId: "abrGvSbKEjM",
+    pages: [1, 2],
+    subtitle: "Filmed in Buenos Aires on May 10, 2024",
+  },
+  {
+    section: "1.3 Scaling Under Size-Synchrony Antagonism",
+    videoId: "8f4aibO5vTI",
+    pages: [2],
+    subtitle: "Filmed in Buenos Aires on May 10, 2024",
+  },
+  {
+    section: "2.1 Polkadot",
+    videoId: "jFUQje4puAM",
+    pages: [2, 3],
+    subtitle: "Filmed in Buenos Aires on May 10, 2024",
+  },
+  {
+    section: "2.2 Ethereum",
+    videoId: "45c2oQTs-P8",
+    pages: [3, 4],
+    subtitle: "Filmed in Buenos Aires on May 10, 2024",
+  },
+  {
+    section: "2.3 Fragmented Meta-Networks",
+    videoId: "AUQZ38Wgl4c",
+    subtitle: "Filmed in Buenos Aires on May 10, 2024",
+    pages: [4, 5],
+  },
+  {
+    section: "2.4 High-Performance Fully Synchronous Networks",
+    videoId: "dhO53bV2b8c",
+    pages: [5],
+    subtitle: "Filmed in Buenos Aires on May 10, 2024",
+  },
+  {
+    section: "3.1 Typography",
+    videoId: "nPKq_5ChZv8",
+    pages: [5],
+    subtitle: "Filmed in Buenos Aires on May 10, 2024",
+  },
+  {
+    section: "3.2 Functions and Operators",
+    videoId: "qOw4XOu_n_w",
+    pages: [6],
+    subtitle: "Filmed in Buenos Aires on May 10, 2024",
+  },
+  {
+    section: "3.3 Sets",
+    videoId: "OqbSSkJFOIk",
+    pages: [6],
+    subtitle: "Filmed in Buenos Aires on May 10, 2024",
+  },
+  {
+    section: "3.4 Numbers",
+    videoId: "9qh4kH6Rewg",
+    pages: [6],
+    subtitle: "Filmed in Buenos Aires on May 10, 2024",
+  },
+  {
+    section: "3.5 Dictionaries",
+    videoId: "JOFb36EJx9Q",
+    pages: [6],
+    subtitle: "Filmed in Buenos Aires on May 10, 2024",
+  },
+  {
+    section: "3.6 Truples",
+    videoId: "irfFhAp3OxA",
+    pages: [6],
+    subtitle: "Filmed in Buenos Aires on May 10, 2024",
+  },
+  {
+    section: "3.7 Sequencers",
+    videoId: "4e1WCtYQbFM",
+    pages: [6, 7],
+    subtitle: "Filmed in Buenos Aires on May 10, 2024",
+  },
+  {
+    section: "3.8 Cryptography",
+    videoId: "2IzvXCE0G5w",
+    pages: [7, 8],
+    subtitle: "Filmed in Buenos Aires on May 10, 2024",
+  },
+  {
+    section: "4 Overview",
+    videoId: "Y4jzP633jiQ",
+    pages: [8],
+    subtitle:
+      "Filmed at the National University of Singapore on June 19 , 2024.",
```

Comment:
Not able to be translated. 🤔 
Maybe just save location and date and use i18next react interpolation feature.

https://www.i18next.com/translation-function/interpolation

Reply:
fixed by adding the key